src/pages/Activity.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react";
+import { fetchActivityByUuid } from "../services/api";
+import { useParams } from "react-router-dom";
+
+export default function Activity() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [selectedActivity, setSelectedActivity] = useState();
+  let { activityUuid } = useParams();
```

Comment:
meglio utilizzare `const` in questo caso, così siamo sicuri che non venga sovrascritto

Reply:
closes #28